docs/code-quality/analyzer-rule-sets.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: FxCop analyzer rule sets and editorconfig files
+title: FxCop analyzer rulesets and editorconfig files
```

Comment:
Neither "ruleset" nor "rule set" are included in Microsoft style guides, so I'd leave as two words, depending on the preference of the content team.

Reply:
Hi, @mavasani, could you double check?

Reply:
Yes, "rule set" is the current spelling in the docs. Please stick with that. Thanks!